include/libXbSymbolDatabase.h

```diff
@@ -127,6 +127,41 @@ static const char* xbe_type_str[XB_XBE_TYPE_MAX] = {
     "CHIHIRO"
 };
 
+typedef enum _XbSDSymbolType {
+    symbol_internal = 0, // Only for internal usage.
+    symbol_variable,
+    symbol_function
+} XbSDSymbolType;
+
+typedef enum _XbSDBParamType {
+    param_unk = 0, // Unknown
```

Comment:
`param_unk` is useless because `XbSymbolDatabase_ParamToString` never checks against it, and instead catches invalid indexes with a `default` keyword. So I suggest this `param_stk = 1,     // 32 bits`

Reply:
Please leave this review remark open, I may had given incorrect comment to the API relative to this.

Reply:
Resolved in updated fixup commit.